LCS.cpp

```diff
@@ -1,4 +1,6 @@
-#include<bits/stdc++.h>
+#include<iostream>
+#include<cstring>
+
```

Comment:
No need to do this

Reply:
I am sorry that was by mistake.
I intended to add the target sum program.